docs/contracts/v4/quickstart/05-subscriber.mdx

```diff
@@ -45,9 +45,11 @@ contract MySubscriber is ISubscriber {
 
 ## 2. A caveat on `unsubscribe()`
 
-There is a variable [`unsubscribeGasLimit`](/contracts/v4/reference/periphery/interfaces/INotifier#unsubscribegaslimit) specifically set at deployment of `PositionManager` to prevent gas griefing on `unsubscribe()` - which could result in the subscriber contract not being notified during `unsubscribe()`. 
+To prevent gas griefing during unsubscription, Uniswap v4 sets a fixed variable [`unsubscribeGasLimit`](/contracts/v4/reference/periphery/interfaces/INotifier#unsubscribegaslimit) when calling a subscriber’s `notifyUnsubscribe()` function.
 
-If notifying the subscriber contract is not necessary users can still specify a gas limit where `notifyUnsubscribe()` hits `OutOfGas` and reverts yet the unsubscription will still succeed.
+Without this limit, if a subscriber's `notifyUnsubscribe()` consumes too much gas, it could cause the unsubscription transaction to revert, preventing the liquidity provider (LP) from exiting the position.
```

Comment:
```suggestion
Without this limit, malicious subscribers could prevent liquidity providers from unsubscribing. If `notifyUnsubscribe()` were to consume too much gas, it would cause the unsubscription transaction to revert, thus leading to a denial-of-service
```